addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -207,8 +256,13 @@ export class AnalyticDistribution extends Component {
         if (this.activeGroup) {
             domain.push(['root_plan_id', '=', this.activeGroup]);
         }
+        return domain;
+    }
 
-        const records = await this.fetchAnalyticAccounts([...domain, '|', ["name", "ilike", request], ['code', 'ilike', request]], 7);
+    async loadOptionsSourceAnalytic(request) {
```

Comment:
`request` is an odd variable name, it makes me think of networking stuff; I'd go for `name` instead